doc/classes/ParticlesMaterial.xml

```diff
@@ -330,5 +333,17 @@
 		<constant name="EMISSION_SHAPE_MAX" value="6" enum="EmissionShape">
 			Represents the size of the [enum EmissionShape] enum.
 		</constant>
+		<constant name="EMISSION_POINTS_ORDER_RANDOM" value="0" enum="PointsOrder">
+			Particles from point textures will be emitted with a random order.
+		</constant>
+		<constant name="EMISSION_POINTS_ORDER_NORMAL" value="1" enum="PointsOrder">
```

Comment:
I'd prefer the name "sequential" as "normal" doesn't really tell what it does just when .reading its name